lib/jekyll-feed/feed.xml

```diff
@@ -33,6 +33,11 @@
 
   {% assign posts = site.posts | where_exp: "post", "post.draft != true" %}
   {% for post in posts limit: 10 %}
+    {% if page.category %}
```

Comment:
We'll need to filter the category above, otherwise the limit will be off. I believe the following should work:

`site.categories[page.category]  | where_exp: "post", "post.draft != true"`